### ISSUE TITLE
Using tween.Complete() after using a loop to modify the Scale value will cause the scale value to be abnormal.

### ISSUE THREAD

**Issue by tpnet** [KyryloKuzyk/PrimeTween#38]
version is 1.1.9,  My code like this:
```
        [SerializeField] private Image mIvCircle;
        private Tween? mTweenCircle;
        public void ShowCircleAni()
        {
            mTweenCircle?.Stop();
            //The original scale size is 1f
            mTweenCircle = Tween.Scale(mIvCircle.rectTransform, 0.7f, 0.5f, cycles: -1, cycleMode: CycleMode.Yoyo);
            Test().Forget();
        }


        /// <summary>
        /// Test Complete
        /// </summary>
        private async UniTask Test()
        {
            await UniTask.Delay(2000);
            //TODO：  Will cause the size of the control to become 1.9f 
            mTweenCircle?.Complete();
        }
```

![image](https://github.com/KyryloKuzyk/PrimeTween/assets/17380722/43eed037-07d8-47bf-a2ab-83b4587fd02e)

**Comment by KyryloKuzyk** [comment]
Huge thanks for the bug report! 
I reproduced it on my side and will get back with a fix soon.

**Comment by KyryloKuzyk** [comment]
@tpnet I fixed the issue in version **1.1.10**. The bug wasn't specific to a Scale() tween, but happened with all tween types. 

Before the fix, PrimeTween tried to complete all remaining cycles and wrongly calculated the interpolation factor for infinite tweens (cycles == -1). 

Now PrimeTween completes only the current tween cycle. So if the tween is playing backward (odd cyclesDone with CycleMode.Yoyo/Rewind) and tween.Complete() is called, the animated value will be set to 'startValue'. To choose between 'startValue' and 'endValue' in the case of infinite cycles, use SetRemainingCycles(bool stopAtEndValue) before calling Complete().

Thanks again for reporting the issue!
